docs/source/man.rst

```diff
@@ -713,7 +753,4 @@ Authors
 
 The Quipucords tool was originally written by Chris Hambridge <chambrid@redhat.com>, Kevan Holdaway <kholdawa@redhat.com>, Ashley Aiken <aaiken@redhat.com>, Cody Myers <cmyers@redhat.com>, and Cecilia Carter <cecarter@redhat.com>.
 
-Copyright
----------
-
-Copyright 2018-2019 Red Hat, Inc. Licensed under the GNU Public License version 3.
```

Comment:
we dropped headers from source code. manpage is a different beast.

Reply:
Earlier commits dropped the license info from source code file headers. Since the man page is user-facing, though, perhaps it needs to keep a copyright statement. I do not have _strong_ opinions here, but _I think_ I would err on the preference to keep it (and update the year… yikes!).

Reply:
I don't see a reason to not follow the same pattern.

Reply:
Alright!

Reply:
The only problem (and we run into this in docs too) is to have people remember to update the copyright year when the year turns over. Is that something we can build in a reminder of? How many man pages are we updating, exactly?

Reply:
Mmmm we could have that automated 🤔 

What Y'all think about adding jinja or some other templating lang to this rst file?
rendering the proper markdown could be done before invoking pandoc to convert it to ascidoc.

Asking because this could also help with the upcoming story for "qpc/dsc" branding.

Reply:
Is the automation for this coming in a _future_ PR? I don't think we need to hold _this_ PR for automation. Just fix it manually for now and track the automation effort elsewhere later.

Reply:
👍  adding jinja support and updating the rst for this bit in a separate PR.

Reply:
yes automation is coming on separate pr